File: C#/FW/FW/DirectedCut.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Onigiri.FW
{
    public class DirectedCut : Cut
    {
        public DirectedCut(int n, double[] modular, double[][] weight)
            : base(n, modular, weight)
        {
        }

        public DirectedCut(string path)
            : base(path)
        {
        }
    }
}

File: C++/SubmodularFunctionOld/SubmodularFunction/DirectedCut.cpp

#include "Submodular.h"

using namespace OnigiriSubmodular;

DirectedCut::DirectedCut(int n,double* modular,double** weight):
	Cut(n,modular,weight)
{
}

DirectedCut::DirectedCut(string path):
	Cut(path)
{
}
